components/gh_events/GitHubEvent.tsx

```diff
@@ -120,45 +143,48 @@ export default function GitHubEvent({ event }: { event?: IGitHubEvent }) {
 
     case "ForkEvent":
       title = (
-        <>
+        <div className="flex gap-1 items-center">
+          <GoRepoForked className="text-blue-500 text-lg font-bold" />
```

Comment:
Let's keep the colors of the icon the same or in line with the theme. We are not using this blue anywhere else IIRC.

I'd prefer to go with the same color as the title text.

Reply:
Okay i understood, i changed the color because in the issue #163 you provide one image so i changed the color. I will change to the same as text color.

Reply:
Okay 😅 My bad, I just took a screenshot from other tool to show as an example

Reply:
@rithviknishad  I removed colors from your icons .Please, review it